src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -839,11 +842,13 @@ const _createSubmission = async ({
     timestamp: createdTime.getTime(),
   })
 
+  // send respondent copy email to respondentEmails
```

Comment:
Nit: I wonder if this comment should be placed here - as it might give the impression that the post actions only does 1 thing which is send respondent copy. 

Shall we place this in the post actions function itself - if the comment is deemed necessary.